src/error.ts

```diff
@@ -174,3 +174,10 @@ export class DBOSDeadLetterQueueError extends DBOSError {
     super(`Workflow ${workflowUUID} has been moved to the dead-letter queue after exceeding the maximum of ${maxRetries} retries`, DeadLetterQueueError);
   }
 }
+
+const FailedSqlTransactionError = 19;
+export class DBOSFailedSqlTransactionError extends DBOSError {
+  constructor(workflowUUID: string, txnName: string) {
+    super(`Postgres aborted the transaction ${txnName} of Workflow ${workflowUUID}.`, FailedSqlTransactionError);
```

Comment:
I think we need to make sure this warning is loud and clear, so maybe also add a `console.error()` here to log the message?

For example, if someone writes a try...catch block that ignores the error, then they won't know what happened...

Reply:
I don't think the call should be in the error ctor, but I will add a error log where the error is thrown